src/components/IndividualPageContent/ContentRow.tsx

```diff
@@ -18,7 +18,7 @@ export default function ContentRow({
   value,
   tooltip,
   i,
-  container,
+  container = true,
```

Comment:
this is to set the container boolean default to true so that if no `container` pass in, it will still have `container={true}`. goal is to fix the current misalignment on the UI, so that it will be inline instead of in two lines.
<img width="469" alt="Screenshot 2023-01-24 at 2 51 13 PM" src="https://user-images.githubusercontent.com/121921928/214438751-50691368-8dad-4496-b70d-0dbeebf789ff.png">